src/applications/claims-status/actions/index.js

```diff
@@ -430,32 +430,35 @@ export function submitRequest(id) {
   };
 }
 
-export const submit5103 = id => {
-  return dispatch => {
-    dispatch({
-      type: SUBMIT_DECISION_REQUEST,
-    });
-
-    makeAuthRequest(
-      `/v0/benefits_claims/${id}/submit5103`,
-      { method: 'POST' },
-      dispatch,
-      () => {
-        dispatch({ type: SET_DECISION_REQUESTED });
-        dispatch(
-          setNotification({
-            title: 'Request received',
-            body:
-              'Thank you. We have your claim request and will make a decision.',
-          }),
-        );
-      },
-      error => {
-        dispatch({ type: SET_DECISION_REQUEST_ERROR, error });
-      },
-    );
-  };
-};
+export const submit5103 = submitRequest();
```

Comment:
Does this need to include an `id`?

Reply:
Oops, I don't want to call `submitRequest` here. Good catch!